features/api_magic_attach.feature

```diff
@@ -1,11 +1,9 @@
 Feature: Magic Attach endpoints
 
-    @series.lts
```

Comment:
I think we should keep the series, right?

Reply:
Yes, my mistake